zpages/templates.go

```diff
@@ -80,22 +81,19 @@ func countFormatter(num int) string {
 	return fmt.Sprint(num)
 }
 
-func msFormatter(d time.Duration) string {
-	if d == 0 {
-		return "0"
-	}
-	if d < 10*time.Millisecond {
-		return fmt.Sprintf("%.3f", float64(d)*1e-6)
+func msFormatter(latency float64) string {
+	if latency < 10.0 {
+		return fmt.Sprintf("%.2f", latency)
 	}
-	return strconv.Itoa(int(d / time.Millisecond))
+	return strconv.Itoa(int(latency))
```

Comment:
This change doesn't make sense to me, in this case why wouldn't we instead just `fmt.Sprintf` as we have right above